modules/salt/pages/formula-yomi.adoc

```diff
@@ -0,0 +1,153 @@
+[[yomi-formula]]
+= Yomi Formula
+
+Yhe Yomi (yet one more installer) installer for {suse} and openSUSE operating systems is configured using formulas with forms.
+
+The ``yomi-formula`` package provides these formulas:
+
+* Yomi
+* Yomi Storage
+* Yomi Bootloader
+* Yomi Software
+* Yomi Services
+* Yomi Users
+
+
+.Procedure: Install the Yomi Formulas with Forms
+. On the {productname} Server, at the command prompt, as root, install the ``yomi-formula`` package:
++
+----
+zypper in yomi-formula
+----
+. In the {susemgr} {webui}, open the details page for the Salt master, and navigate to the Formulas tab.
+. Check the [guimenu]``Installer`` box, and click btn:[Save].
+. Apply the highstate.
+
+
+
+.Procedure: Configuring the Yomi Formula
+. Navigate to the menu:Formulas[Yomi] tab, and set these parameters in the [guimenu]``General Configuration`` section:
+* Check the [guimenu]``Events`` box to allow monitoring.
+* In the [guimenu]``Reboot`` field, select [systemitem]``yes`` to instruct the client to reboot after installation.
+* Check the [guimenu]``Snapper`` box if you are using the btrfs file system on the client.
+* In the [guimenu]``Locale`` field, select the region and encoding for systemd to use on the client (for example: [systemitem]``en_US.utf8`` for American English and UTF-8).
+* In the [guimenu]``Keymap`` field, select the appropriate keyboard layout (for example: [systemitem]``us`` for an American keyboard layout).
+* In the [guimenu]``Timezone`` field, select the timezone for the client to use (for example: [guimenu]``America/New_York`` for EST).
+* In the [guimenu]``Hostname`` field, enter the hostname for the client to use.
+Leave this blank if you are using DHCP to provide the hostname.
+* In the [guimenu]``Machine Id`` field, enter a machine identification number for the client.
+Leave this blank to have systemd generate one automatically.
+* In the [guimenu]``Target`` field, enter a systemd target unit.
+. Click btn:[Save Formula] to save your configuration.
+. Apply the highstate.
+
+
+
+.Procedure: Configuring the Yomi Storage Formula
+. Navigate to the menu:Formulas[Yomi Storage] tab, and set these parameters in the menu:Partitions[Config] section:
+* In the [guimenu]``Labels`` field, select the default partition table type to use.
+* In the [guimenu]``Initial Gap`` field, select the default amount of space to leave before each partition.
+For example: [systemitem]``1{nbsp}MB``, or use [systemitem]``0`` to leave no space between partitions.
+. For each device that you want to create, in the menu:Partitions[Devices] section, click btn:[+], and set these parameters:
```

Comment:
for each device that you want to configure (or allocate, or partition). The devices are already in the hardware, the user cannot create them.